new-frontiers-14/frontier-station-14
# Sergeant and Bailiff ship vouchers, round-start NFSD and pirate roles

## About the PR



This PR adds round-start slots for the NFSD and Pirates (as of writing, these are 1 bailiff slot, 1 sergeant slot, 2 deputy slots, 1 pirate captain slot and 1 pirate first mate slot).

Bailiffs and sergeants receive vouchers in their loadout to exchange for free ships.  This voucher stores the ship's registration, and is destroyed on sale.

Also, fixed some minor inconsistencies in NFSD ID cards and job icons.

## Why / Balance


Absence of NFSD presence prevents pirate roles from opening, this allows for a more complete game loop.

## How to test


1. Join as a pirate! (Whitelist yourself through `whitelistadd <yourusername>`)
2. Join as a Bailiff, use your ship voucher to receive a free ship (small to capital) from the NFSD shipyard terminal.
3. Sell your ship.  Your voucher should disappear, like dust in the wind.
4. Join as a Sergeant, use your ship voucher to receive a free ship (small to medium) from the NFSD shipyard terminal.
5. Sell your ship.  Your voucher should disappear, like dust in the wind.

## Media

Round start NFSD slots:
![round-start-slots](https://github.com/user-attachments/assets/654e3593-9d3e-4942-8b60-497e90c7ffed)
Vouchers:
![image](https://github.com/user-attachments/assets/62dcb42b-de6f-4182-8fa2-c3eb19921618)
Shipyard with vouchers:
![shipyard](https://github.com/user-attachments/assets/c3415833-5a7f-44bc-aa48-79f88c947aee)


## Breaking changes


**Changelog**



:cl: whatston3 and Salvantrix
- add: Pirates and NFSD now have more round-start job slots.
- add: Sergeants and Bailiffs start with ship vouchers that allow for free ship purchase & sale.
